ppb/events.py

```diff
@@ -146,108 +218,129 @@ class ReplaceScene:
     Examples:
         * `signal(new_scene=ReplaceScene(MyScene(player=player))`
         * `signal(new_scene=ReplaceScene, kwargs={"player": player}`
+
+    .. warning::
+       In general, you should not respond to ReplaceScene, if you want to
+       respond to a new scene, see :class:`SceneStarted`.
     """
-    new_scene: Union[BaseScene, Type[BaseScene]]
-    kwargs: Dict[str, Any] = None
-    scene: BaseScene = None
+    new_scene: Union[BaseScene, Type[BaseScene]]  #: A :class:`~ppb.scenes.BaseScene` class or instance
+    kwargs: Dict[str, Any] = None  #: Keyword arguments to be passed to a scene type.
+    scene: BaseScene = None  #: The currently running scene.
 
 
 @dataclass
 class SceneContinued:
     """
-    Fired when a paused scene continues.
+    A scene that had been paused has continued.
 
     This is delivered to a scene as it resumes operation after being paused via
     a ScenePaused event.
 
-    From the middle of the event lifetime that begins with SceneStarted.
+    From the middle of the event lifetime that begins with
+    :class:`SceneStarted`.
     """
-    scene: BaseScene = None
+    scene: BaseScene = None  #: The currently running scene.
 
 
 @dataclass
 class SceneStarted:
     """
-    Fired when a scene starts.
+    A new scene has started running.
 
     This is delivered to a Scene shortly after it starts. The beginning of the
-    scene lifetime, ended with SceneStopped, paused with ScenePaused, and
-    resumed from a pause with SceneContinued.
+    scene lifetime, ended with :class:`SceneStopped`, paused with
+    :class:`ScenePaused`, and resumed from a pause with :class:`SceneContinued`.
     """
-    scene: BaseScene = None
+    scene: BaseScene = None  #: The currently running scene.
```

Comment:
Just in the various lifecycle events, or. . . all of them?

Reply:
Specifically in the lifecycle events, because they represent the boundary between events.

Something like "the scene that is starting" or "the scene that was paused"